README.md

```diff
@@ -44,6 +44,88 @@ docker build -t pca-assessment .
 We welcome contributions!  Please see [here](CONTRIBUTING.md) for
 details.
 
+## Assessment JSON Field Dictionary ##
+
+The following items are included in the assessment JSON as produced by the `pca-wizard`.
+An example assessment JSON can be found [here](src/assessment/sample_assessment.json).
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| id | Assessment identifier. (e.g. "RV0000") | string | | yes |
+| timezone | Timezone name based on [pytz](http://pytz.sourceforge.net/) timezones. (e.g. "US/Eastern") | string | | yes |
+| domain | Assessment domain for GoPhish public interface. (e.g. "domain.tld") | string | | yes |
+| target_domain | Approved target domains where all email recipients must reside. (e.g. ["target1.tld", "target2.tld"]) | list(string) | | yes |
+| start_date | Assessment start date in 24-hr ISO format with offset. (e.g. "2020-01-01T14:00:00-04:00") | string | | yes |
+| end_date | Assessment end date in 24-hr ISO format with offset. (e.g. "2020-01-06T15:30:00-04:00") | string | | yes |
+| reschedule | Indicates if the assessment json is a rescheduled assessment. | boolean | | yes |
+| start_campaign | The campaign that the assessment should start at. | integer | `1` | no |
+| groups | Consolidated list of email recipients grouped to receive campaigns, [example](#group-dictionary). | list(dictionaries) | | yes |
```

Comment:
Sorry, I just noticed this one (occurs in multiple places):
`list(dictionaries)` -> `list(dict)`